### PR TITLE
Tmz bugfix wig issue 1710

### PR DESCRIPTION
**Pull request by t-m-z**
check for and react on multiple deobfuscation functions

### CODE REVIEW

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 1]
```diff
@@ -1858,19 +1848,19 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
         LUAFile = LUAFile.replaceAll(key, 'objVariable_' + key);
       });
 
-      RegExp(r'deObfuscate\(".*?"\)').allMatches(LUAFile).forEach((obfuscatedText) {
-        var group = obfuscatedText.group(0);
-        if (group == null) return;
-        LUAFile = LUAFile.replaceAll(group, _deObfuscate(group));
-      });
+      // RegExp(r'deObfuscate\(".*?"\)').allMatches(LUAFile).forEach((obfuscatedText) {
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1851]: clean code

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 2]
```diff
@@ -1858,19 +1848,19 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
...
+      //   var group = obfuscatedText.group(0);
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1852]: clean code

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 3]
```diff
@@ -1858,19 +1848,19 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
...
+      // });
     }
     return LUAFile;
   }
 
-  String _deObfuscate(String obfuscatedText) {
-    obfuscatedText = obfuscatedText.replaceAll('deObfuscate("', '').replaceAll('")', '');
-    return '"' + deobfuscateUrwigoText(obfuscatedText, WherigoCartridgeLUAData.ObfuscatorTable) + '"';
-  }
+  // String _deObfuscate(String obfuscatedText) {
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1860]: clean code

**Review by Mike-3 — CHANGES_REQUESTED** [review]

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 4]
```diff
@@ -1830,7 +1820,7 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
 
   String _normalizeLUA(String LUAFile, bool deObfuscate) {
     if (deObfuscate) {
-      LUAFile = LUAFile.replaceAll(WherigoCartridgeLUAData.ObfuscatorFunction, 'deObfuscate');
+      //LUAFile = LUAFile.replaceAll(WherigoCartridgeLUAData.ObfuscatorFunction, 'deObfuscate');
```
[the remark this answers is not in the document]
Comment (t-m-z) [reply lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1823]: done

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart (hunk 3 above)
Comment (t-m-z) [reply lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1860]: done

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart (hunk 2 above)
Comment (t-m-z) [reply lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1852]: done

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart (hunk 1 above)
Comment (t-m-z) [reply lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1851]: done

**Pull request merged**